docs/whichkey/extra.md

```diff
@@ -44,6 +44,36 @@ You can selectively override that color in your `settings.json` like the followi
 },
 ```
 
+## Styling the menu with Customize UI
```

Comment:
```suggestion
## Style the menu with Customize UI
```
Sorry for a repeated suggestion on the same line. Thanks for updating according to my past suggestion. I deleted my previous suggestion, because it sounded grammatical incorrect.

Reply:
No worries, happens!